ledger/ledger.go

```diff
@@ -323,29 +323,63 @@ func (l *Ledger) GetLastCatchpointLabel() string {
 // GetAssetCreatorForRound looks up the asset creator given the numerical asset
 // ID. This is necessary so that we can retrieve the AssetParams from the
 // creator's balance record.
-func (l *Ledger) GetAssetCreatorForRound(rnd basics.Round, assetIdx basics.AssetIndex) (basics.Address, error) {
+func (l *Ledger) GetAssetCreatorForRound(rnd basics.Round, aidx basics.AssetIndex) (creator basics.Address, ok bool, err error) {
```

Comment:
what are the gains of having 3 functions instead of just one ?
GetAssetCreatorForRound
GetAppCreatorForRound
GetCreatorForRound

it looks to me that it's a lot of pointless interface dups.

Reply:
Yeah, you're right. Removed `GetAssetCreator` and `GetAppCreator` in favor of a generic `GetCreator` that accepts the type as an additional argument. Thanks!